src/utils/api.js

```diff
@@ -0,0 +1,39 @@
+import gameId, { API_URL } from "../globals.js";
+
+const API_FULL_URL = `${API_URL}/${gameId}/scores`;
+
+// eslint-disable-next-line consistent-return
+const getData = async () => {
+  try {
+    const res = await fetch(API_FULL_URL);
+
+    if (!res.ok) throw new Error("Something went wrong");
+    const data = await res.json();
+
+    return data;
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
+export const postData = async (formData) => {
+  try {
+    const res = await fetch(API_FULL_URL, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(formData),
+    });
+
+    if (!res.ok) throw new Error("Something went wrong");
+
+    // const formattedResult = await res.json();
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error);
+  }
+};
+
+export default getData;
```

Comment:
- [ ] Kindly remove all inline comments to follow best practices, it makes the code look messy
- [ ] Please try not do disable the linters, they guard against unforeseen errors